src/lib/wallet.js

```diff
@@ -37,6 +37,36 @@ async function initMiddleware() {
     'names/auctions': {
```

Comment:
Can you remove this definition and use one provided by the middleware team? https://mainnet.aeternity.io/mdw/swagger/swagger.json

Reply:
<img width="565" alt="Screenshot 2020-12-01 at 13 39 07" src="https://user-images.githubusercontent.com/9007851/100730049-99229680-33da-11eb-9501-cd80132b8faa.png">